packages/model/src/logic/Scope.ts

```diff
@@ -87,8 +87,8 @@ export function Scope(subject: Model, options: Scope.ScopeOptions = {}) {
 
     const useCache = options.forceCache || Object.isFrozen(owner);
 
-    let deconflictedMemberCache: Map<Model, Model[]> | undefined;
-    let conformantMemberCache: Map<Model, Model[]> | undefined;
+    let deconflictedMemberCache: Map<Model, Map<Set<ElementTag>, Model[]>> | undefined;
```

Comment:
If I look at the call sites that perform filtering they are potentially generating a Set dynamically...  So this would end up being a memory leak rather than a cache.  Maybe could sort & concatenate the tags to create a string key instead?